src/errors/helper.js

```diff
@@ -178,6 +178,10 @@ module.exports = {
         else if (err.message === 'java.net.ConnectException: Connection refused: connect') {
             return new OxError(ERROR_CODES.WEBDRIVER_ERROR, 'The underlying WebDriver seems to have crashed: ' + err.message);
         }
+        //  no such window, target window already closed, web view not found
```

Comment:
For what browsers or versions could this happen and in what situations? This should be commented.

Reply:
chrome 86

Reply:
"when doing an operation on an element (e.g. click) but the window was already closed"

Reply:
Fixed